rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -591,7 +621,8 @@ boolean isStaled(long now) {
     @Override
     void cleanup() {
       synchronized (lock) {
-        refCountedChildPolicyWrapperFactory.release(childPolicyWrapper);
+        for (ChildPolicyWrapper policyWrapper : childPolicyWrappers)
```

Comment:
[Our style guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used) always uses braces for `for` blocks.